### DISCUSSION TITLE
Remove underline style in weekdays label

### DISCUSSION THREAD

**Discussion by el0i404** [wojtekmaj/react-calendar#936] [Q&A] [answered]
I ran into this issue, I'm not able to disable this underline on the weekdays label. I've tried with `text-decoration:none` on dev tools in Chrome and in Firefox developer edition but not success. Any idea on how to fix it? Is just a bug?

![Screenshot 2019-08-15 at 19 12 35](https://user-images.githubusercontent.com/4323435/63112500-aa9b2180-bf90-11e9-9115-22d324dd526d.png)

**Comment by wojtekmaj** [comment, marked as answer]
It's not a bug, it's an [accessibility feature](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr). `text-decoration: none;` 100% works if you target proper element (`.react-calendar__month-view__weekdays__weekday abbr`).

**Comment by el0i404** [comment]
@wojtekmaj btw I have to say you made a really good job with this calendar, REALLY well documented, congrats! 

THANKS :)

p.s Also would like to point It's a really lightweight one just `2kb` minified.

<img width="1671" alt="Screenshot 2019-08-17 at 13 29 41" src="https://user-images.githubusercontent.com/4323435/63211127-72f8ba80-c0f3-11e9-9c77-2ea972f43d3b.png">

**Comment by wojtekmaj** [comment]
I don't think it's 2 KB :D It's half the size of other popular solutions, but definitely not 2 KB.

**Comment by ghost** [comment]
Did you solve it?

**Comment by JP-Soup** [comment]
I can confirm that the solution provided works. Great job, thank you.

**Comment by nezeracuk** [comment]
> It's not a bug, it's an [accessibility feature](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr). `text-decoration: none;` 100% works if you target proper element (`.react-calendar__month-view__weekdays__weekday abbr`).

where do I need to enter this parameter to make it work for the calendar?

**Reply by xuzhihuiCSY** [reply]
.react-calendar__month-view__weekdays__weekday abbr{
  text-decoration: none;
}

**Reply by nezeracuk** [reply]
thank you very much!
